mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -0,0 +1,70 @@
+config_opts['root'] = 'fedora-eln-{{ target_arch }}'
+
+config_opts['chroot_setup_cmd'] = 'install @build'
+
+config_opts['dist'] = 'eln'  # only useful for --resultdir variable subst
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['package_manager'] = 'dnf'
+config_opts['bootstrap_image'] = 'fedora:latest'
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+install_weak_deps=0
+metadata_expire=0
+best=1
+
+# TODO
+module_platform_id=platform:eln
```

Comment:
@sgallagher is this correct?

Reply:
The modularity side of things is still being figured out. Let me get back to you on that tomorrow, please.

Reply:
Gently ping here.